android/src/main/java/com/armsaudio/ArmsaudioModule.kt

```diff
@@ -308,22 +308,14 @@ class ArmsaudioModule(reactContext: ReactApplicationContext) :
 
     @ReactMethod
     fun setAudioProgress(progress: Double, promise: Promise) {
-        // Pause the mix
-        if (!isMixPaused) pauseResumeMix()
-
-        // Seek to the new position
         setPosition(progress.toFloat())
 
         promise.resolve(true)
     }
 
     @ReactMethod
     fun audioSliderChanged(progress: Double, promise: Promise) {
```

Comment:
Probably we don't need one of these methods if they are the same

Reply:
@svetlana-zhuravitskaya yes, please don't use setAudioProgress for now

Reply:
@ka8725 Maybe it would be better to remove `audioSliderChanged`? `setAudioProgress` seems like a more suitable method name.

Reply:
No, there is also iOS version